src/main/java/kr/allcll/checkll/api/AdminService.java

```diff
@@ -0,0 +1,26 @@
+package kr.allcll.checkll.api;
+
+import kr.allcll.checkll.datasource.PastGradeDataSource;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdminService {
+
+    private final PastGradeDataSource pastGradeDataSource;
+
+    public AdminService(PastGradeDataSource pastGradeDataSource) {
+        this.pastGradeDataSource = pastGradeDataSource;
+    }
+
+    public void setUpInfoByString(String credentialInfo) {
+        pastGradeDataSource.setCredential(credentialInfo);
+    }
```

Comment:
R: 메서드명에서 ByString은 불필요하다고 생각해요. info도요! 

```suggestion
    public void setUpInfo(String credential) {
        pastGradeDataSource.setCredential(credential);
    }
```

Reply:
그런데 저 setUpInfo에서 Info는 수정할 필요가 없을까요?

Reply:
오.. 좋은데요? 수정해 주세요!